RubeesEat/Pages/ShowBills.cshtml

```diff
@@ -46,18 +46,18 @@
     <div class="pagination justify-content-center">
         @if (billsPaginated.HasPreviousPage)
         {
-            <a href="../ShowBills/?page=@(page - 1)" t-action="prevButton" t-click-behavior="Click">Prev</a>
+            <a href="../ShowBills/?page=@(page - 1)" t-action="prevButton" t-click-behavior="Click">Zurück</a>
         }
 
-        <span class="pageInfo">Viewing page @(page)/@billsPaginated.TotalPages</span>
+        <span class="pageInfo"> Seite @(page)/@billsPaginated.TotalPages</span>
 
         @if (billsPaginated.HasNextPage)
         {
-            <a href="../ShowBills/?page=@(page + 1)" t-action="nextButton" t-click-behavior="Click">Next</a>
+            <a href="../ShowBills/?page=@(page + 1)" t-action="nextButton" t-click-behavior="Click">Nächste</a>
```

Comment:
The pair would be "Vor"/"Zurück" or "Vorhige Seite"/"Nächste Seite". Right now, it seems a bit inconsistent.